EventFilter/CSCRawToDigi/src/CSCTMBHeader.cc

```diff
@@ -174,14 +174,14 @@ void CSCTMBHeader::selfTest() {
       // In 2007 firmware, there are no distrips, so the 4th argument (strip
       // type) should always be set to 1 (halfstrips).
       CSCCLCTDigi clct0(1, 1, 4, 1, 0, 30, 4, 2, 1);  // valid for 2007
-      CSCCLCTDigi clct1(1, 1, 2, 1, 1, 31, 1, 2, 2);
+      CSCCLCTDigi clct1(1, 1, 3, 1, 1, 31, 1, 2, 2);
```

Comment:
(Just checking) Since all other parameters in this vector as well as in the other ones remain the same, and only the third one in "clct1" gets updated ("pattern" changed from "2" to "3"), please confirm that this is actually wanted.

Reply:
That is correct. Pattern Id 2 with bending 1 is not valid; pattern Id 3 with bending 1 is.

Reply:
So this also looks like a bug fix...

Reply:
Yes. But it did not show up because the `CSCCLCTDigi::operator==` was not sensitive to it